api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -2745,6 +2745,13 @@ message AppSessionStart {
     (gogoproto.embed) = true,
     (gogoproto.jsontag) = ""
   ];
+
+  // AWS contains common AWS session information.
+  AWSSessionMetadata AWS = 9 [
```

Comment:
Can we make it more generic like 
```suggestion
  AWSMetadata AWS = 9 [
```

I wonder what other fields can be added to AWSSessionMetadata, where AWSMetadata would be more flexible and in the case of other AWS fields not related to AWSSession context  I think that  the  AWSMetadata it will be a better fit.